react-bootstrap/react-bootstrap.d.ts

```diff
@@ -128,7 +128,7 @@ declare module "react-bootstrap" {
     interface MenuItemClass extends React.ComponentClass<MenuItemProps> { }
     var MenuItem: MenuItemClass;
 
-
+	type renderable = string | number | Element;
```

Comment:
@myitcv This has now been changed to one of the react based types `ReactNode` which covers all the types mentioned in the document.